src/current/v23.1/drop-schedules.md

```diff
@@ -11,6 +11,8 @@ docs_area: reference.sql
 `DROP SCHEDULE` does **not** cancel any in-progress jobs started by the schedule. Before you drop a schedule, [cancel any in-progress jobs]({% link {{ page.version.version }}/cancel-job.md %}) first, as you will not be able to look up the job ID once the schedule is dropped.
 {{site.data.alerts.end}}
 
+When `DROP SCHEDULES` removes a [scheduled full backup]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), all associated [scheduled incremental backups]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules) are dropped. 
```

Comment:
the backup itself is not dropped, rather the schedule is dropped

I'd rewrite this as "When `DROP SCHEDULES` removes a full backup schedule, it removes the associated incremental backup schedule, if it exists"